src/components/modals/authentification/ModalAuthInitView.tsx

```diff
@@ -0,0 +1,123 @@
+                fill="#34A853"
+              />
+              <path
+                d="M56.281 156.37c-2.756-8.123-4.351-16.827-4.351-25.82 0-8.994 1.595-17.697 4.206-25.82l-.073-1.73L15.26 71.312l-1.335.635C5.077 89.644 0 109.517 0 130.55s5.077 40.905 13.925 58.602l42.356-32.782"
+                fill="#FBBC05"
+              />
+              <path
+                d="M130.55 50.479c24.514 0 41.05 10.589 50.479 19.438l36.844-35.974C195.245 12.91 165.798 0 130.55 0 79.49 0 35.393 29.301 13.925 71.947l42.211 32.783c10.59-31.477 39.891-54.251 74.414-54.251"
+                fill="#EB4335"
+              />
+            </svg>
+          </span>
+          <span className="text-vintedTextGrisClair">
+            Continuer avec Google
+          </span>
+        </button>
+
+        <button
+          type="button"
+          role="button"
+          data-testid="button-apple"
+          className="flex justify-center items-center h-11 w-full border border-solid border-vintedTextGrisClair rounded mt-2 mb-6 cursor-not-allowed"
+          disabled
+        >
+          <span className="pr-[6px]">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 24 24"
+              width="24px"
+              height="24px"
+            >
+              {' '}
+              <path d="M 16.125 1 C 14.972 1.067 13.648328 1.7093438 12.861328 2.5273438 C 12.150328 3.2713438 11.589359 4.3763125 11.818359 5.4453125 C 13.071359 5.4783125 14.329031 4.8193281 15.082031 3.9863281 C 15.785031 3.2073281 16.318 2.12 16.125 1 z M 16.193359 5.4433594 C 14.384359 5.4433594 13.628 6.5546875 12.375 6.5546875 C 11.086 6.5546875 9.9076562 5.5136719 8.3476562 5.5136719 C 6.2256562 5.5146719 3 7.4803281 3 12.111328 C 3 16.324328 6.8176563 21 8.9726562 21 C 10.281656 21.013 10.599 20.176969 12.375 20.167969 C 14.153 20.154969 14.536656 21.011 15.847656 21 C 17.323656 20.989 18.476359 19.367031 19.318359 18.082031 C 19.922359 17.162031 20.170672 16.692344 20.638672 15.652344 C 17.165672 14.772344 16.474672 9.1716719 20.638672 8.0136719 C 19.852672 6.6726719 17.558359 5.4433594 16.193359 5.4433594 z" />
+            </svg>
+          </span>
+          <span className="text-vintedTextGrisClair">Continuer avec Apple</span>
+        </button>
+        <p className="text-vintedTextBlack pb-2">
+          Ou connecte toi avec{' '}
+          <span
+            className="text-vintedGreen cursor-pointer"
+            onClick={() => setContenuModal('connexion')}
+          >
+            e-mail
+          </span>
+        </p>
+        <p className="text-vintedTextBlack text-center">
+          Tu n'as pas de compte Vinted ?{' '}
+          <span
+            className="text-vintedGreen cursor-pointer"
+            onClick={() => setContenuModal('inscription')}
```

Comment:
2 messages de Sonarlint :
- Avoid non-native interactive elements. If using native HTML is not possible, add an appropriate role and support for tabbing, mouse, keyboard, and touch inputs to an interactive content element.sonarlint(typescript:S6848)
- Visible, non-interactive elements with click handlers must have at least one keyboard listener.sonarlint(typescript:S1082)

Reply:
idem role="button"